programs/zstdcli.c

```diff
@@ -637,6 +637,11 @@ static void printVersion(void)
     #endif
         DISPLAYOUT("\n");
         if (g_displayLevel >= 4) {
+            /* library versions */
+            DISPLAYOUT("zlib version %s\n", FIO_zlibVersion());
```

Comment:
Do I understand that a library version is displayed for each and every supported external codec,
irrespective of the fact that `zstd` was linked to them or not ?
(and display "Unsupported" in this last case)

Reply:
Yup. We could change that behavior by gating each of these behind an `ifdef` check. I figured that since this is in `zstd -vvV` output, there is not need to make it super pretty. I'm okay either way though, so lmk if you want me to change it.

Reply:
It's not urgent indeed. This is a debug print, I think some verbosity is acceptable.